pegasus/migrations/096_drop_unique_constraint_from_forms_source_id_index.rb

```diff
@@ -0,0 +1,22 @@
+Sequel.migration do
+  up do
+    # For some reason this unique index only exists on production, but not staging or test.
+    # Conditionally drop it if it exists.
+    if self.indexes(:forms)[:unique_forms_source_id]
+      alter_table(:forms) do
+        drop_index :source_id, name: :unique_forms_source_id
+      end
+    end
+
+    # Replace with a unique index on source_id and kind (rather than just source_id).
+    alter_table(:forms) do
+      add_index [:source_id, :kind], unique: true
+    end
+  end
```

Comment:
My (uncertain) belief is that it is a factor of both numbers. Just asking, I don't know what the best course of action is. Note that the pending `contacts` migration won't be short or fast, using the `poste_deliveries` (having an order of magnitude more rows) migration as a benchmark.